python/setup.py

```diff
@@ -71,7 +84,14 @@ def _build_with_cmake(self, ext: CMakeExtension) -> None:
             ["cmake", "--build", ".", *build_args], cwd=build_temp, check=True
         )
 
+
+def version_scheme(version):
+    return get_version("hyperon/_version.py")
```

Comment:
I provided this code as an example. For the real code one need to check whether `version` was detected correctly by checking if it is `None` or `0.0`. If it isn't detected correctly then use hard-coded version.

Reply:
You mean?
```
def version_scheme(version):
    if (version is None) or (version == "0.0"):
        return get_version("hyperon/_version.py")
    else:
        return version
```

Reply:
Taking my words back, in fact we don't want two sources for version: tags and file, it is better to have a single source and I would prefer it would be the file.